lib/sqlalchemy/sql/schema.py

```diff
@@ -1599,6 +1599,15 @@ def get_children(self, schema_visitor=False, **kwargs):
         else:
             return ColumnClause.get_children(self, **kwargs)
 
+    @property
+    def _is_readonly(self):
```

Comment:
we would save a lot of method calls when mappers are set up if it were done this way:

```
class Column(...):
    _is_readonly = False

....

class Computed(...):
   def _set_parent(self, parent):
       # ...
       parent._is_readonly = True


```

Reply:
could't we use `memoized_property` here?

Reply:
sure but it wouldn't change the typical number of method calls, that is, create Table + Columns -> pass to mapper, calls the method on every Column.   "computed" is going to be a vanishingly small minority of mapped columns so I like the idea of it leaving no footprint if not used